var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -52,10 +53,13 @@ class Fftw(Package):
     def install(self, spec, prefix):
         options = ['--prefix=%s' % prefix,
                    '--enable-shared',
-                   '--enable-threads',
-                   '--enable-openmp']
+                   '--enable-threads']
         if not self.compiler.f77 or not self.compiler.fc:
             options.append("--disable-fortran")
+        # Add support for OpenMP
+        # Note: Make sure your compiler supports OpenMP
+        if '+openmp' in spec:
+            options.append('--enable-openmp')
```

Comment:
don't know which one will be merged first, but you could add
```
            # Note: Apple's Clang does not support OpenMP.
            if spec.satisfies('%clang'):
                raise InstallError("Apple's clang does not support OpenMP")
```
before this line to make sure noone tries openmp with clang.